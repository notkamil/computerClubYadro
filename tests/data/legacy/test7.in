2
10:00 12:00
10
10:05 1 client1
10:10 2 client1 1
10:15 3 client1