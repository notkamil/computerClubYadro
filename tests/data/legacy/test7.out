10:00
10:05 1 client1
10:10 2 client1 1
10:15 3 client1
10:15 13 ICanWaitNoLonger!
12:00 11 client1
12:00
1 20 01:50
2 0 00:00